kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/GetTopicConfigResponse.java

```diff
@@ -20,19 +20,19 @@
 import java.util.StringJoiner;
 
 /**
- * Response body for {@code GET /v3/clusters/<clusterId>/topics/<topicName>/configurations/<name>}
+ * Response body for {@code GET /v3/clusters/<clusterId>/topics/<topicName>/configs/<name>}
```

Comment:
`Response body for {@code GET /v3/clusters/<clusterId>/topics/<topicName>/config/<name>}`